tests/unit/date/helper.js

```diff
@@ -0,0 +1,30 @@
+define( [
+	"jquery",
+	"globalize",
+	"globalize/date",
+	"lib/helper"
+], function( $, Globalize, helper ) {
+
+return $.extend( helper, {
+	getAttributes: function( locale ) {
```

Comment:
@rxaviers Since we now need to pass attributes parameter I adopted what we do in calendar itself. Is this a good solution or is there a better way to do so?